src/OrchardCore.Modules/OrchardCore.PublishLater/Drivers/PublishLaterPartDisplayDriver.cs

```diff
@@ -49,7 +48,7 @@ public override async Task<IDisplayResult> UpdateAsync(PublishLaterPart part, IU
         {
             var httpContext = _httpContextAccessor.HttpContext;
 
-            if (await _authorizationService.AuthorizeAsync(httpContext?.User, Permissions.PublishContent, part.ContentItem))
+            if (await _authorizationService.AuthorizeAsync(httpContext?.User, CommonPermissions.PublishContent, part.ContentItem))
```

Comment:
I see, I didn't know this was refectored.

Reply:
The rule is that a module should not reference a module. That's why it was done this way.